lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -393,7 +408,7 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
     ),
   );
 
-  $custom_share_link = $campaign_path . '?source=' . $vars['user']->uid;;
+  $custom_share_link = $campaign_path . '?source=' . $vars['user']->uid;
```

Comment:
I didn't catch this before but I think we want the `source` to be `source=user/[uid]`